### DISCUSSION TITLE
Error Connecting (500)

### DISCUSSION THREAD

**Discussion by helpplease12344** [DaxStudio/DaxStudio#972] [Q&A]
Hello,
I have been using Dax Studio for a long time for power pivot models. The connection was perfect.
I haven't been able to log in for 2 months. "Error connecting" the remote server returned an error, the connection timed out or was lost.
What do I need to do to make this work again?
Thank you very much

**Comment by helpplease12344** [comment]
here is what is displayed


![Connection1](https://user-images.githubusercontent.com/115630943/195329307-259b98f6-06db-4b15-b92f-c3654ed8571f.png)
![Connection2](https://user-images.githubusercontent.com/115630943/195329333-a676bd95-b970-41cc-971a-3cd1a7e1c3c4.png)

**Comment by dgosbell** [comment]
This is an excel bug that was introduced in the August updates for Excel. I'm tracking in issue #884 - The Excel team have a fix for this we are just waiting for it to go through security and quality reviews. Once the fix is out DAX Studio should start working again.

**Reply by helpplease12344** [reply]
Thank you very much for you answer
